src/scripts/number-to-english.js

```diff
@@ -0,0 +1,117 @@
+// copy-paste from https://maruzzing.github.io/study/algorithm/numberToEnglish/
```

Comment:
Seriously?
It would be better to use something like this https://www.npmjs.com/package/number-to-words